### DISCUSSION TITLE
Elección competencia Kaggle

### DISCUSSION THREAD

**Discussion by malalalalala** [rramosp/ai4eng.v1#268] [00 - General]
Buenas tardes profe, mi nombre es Laura Ramos y la semana pasada estuvimos revisando varias competencias para revisar si alguna se ajustaba a los parámetros del proyecto de la materia.

Después de mucho buscar encontré esta competencia que busca predecir del precio de las viviendas de acuerdo a una serie de características.

https://www.kaggle.com/competitions/house-prices-advanced-regression-techniques/data?select=train.csv

Yo descargué las bases de datos y tiene 81 columnas 1500 datos para testear y 1500 para entrenar al modelo. Es posible que utilicemos esta competencia? Quedo atenta y muchas gracias.

**Comment by Felipe-RA** [comment]
Saludos @lauramosdim 

Los invito a buscar otro dataset ya que este se trabaja en los vídeos. 

[04.01 - DATA EXPLORATION](https://rramosp.github.io/ai4eng.v1/content/NOTES%2004.01%20-%20DATA%20EXPLORATION.html)

¡Hasta pronto!

**Reply by malalalalala** [reply]
Gracias Feli, estas dos competencias son las que le mostré hoy al profesor en la clase presencial:

https://www.kaggle.com/competitions/fcg-2022-netherlands-accommodation-prices/data?select=test.csv
https://www.kaggle.com/competitions/fcg-2022-netherlands-accommodation-prices/data?select=test.csv

en un principio parecería que las dos cumplen, cuál recomiendas o qué parámetros tenemos en cuenta para seleccionar una?

**Reply by Felipe-RA** [reply]
Hola @lauramosdim 

**Pusiste dos veces el mismo enlace!!!!**

Esta competencia cumple con los requisitos y además presenta retos interesantes en cuanto a decisiones de limpieza de datos.

Eso si, varias de las columnas tienen información que no es trivial procesar inicialmente (enlaces a imágenes, texto plano descriptivo), cuando estén procesandolos tendran que tomar decisiones sobre qué hacer con estas.

Si deciden irse por este dataset, tendran que trabajarle un poco más que con otros. 

Pero cumple los requisitos del curso.

**Reply by malalalalala** [reply]
jajaja perdón @Felipe-RA , esta era la otra competencia: https://www.kaggle.com/competitions/forest-cover-type-prediction/data
